lib/appeals_status/configuration.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+require 'common/client/configuration/rest'
+require 'common/client/middleware/request/camelcase'
+require 'common/client/middleware/response/json_parser'
+require 'common/client/middleware/response/raise_error'
+require 'common/client/middleware/response/snakecase'
+require 'typhoeus'
+require 'typhoeus/adapters/faraday'
```

Comment:
Looks like you're requiring typhoeus, but it appears as though you just use the default Faraday adapter instead of typhoeus.